### ISSUE TITLE
Add some "economic performance indicators" graph

### ISSUE THREAD

**Issue by ThePiachu** [StrangeLoopGames/EcoSuggestions#1266]
It might be interesting to develop some economic performance indicators to be able to see at a glance how well a server is performing economically. Those could focus on some key products in the game and hopefully let people know which professions need picking up.

The measurement could be something like iron production / availability, food quality availability, lumber production / availability, money velocity, or the like. Might be worth asking an economist what might be some good measures.

Ideally, those would show you how advanced the server is - hewn production should be a good indicator of T1, iron and lumber could show how well T2 is progressing, oil production could be an indicator for T3 or something like that. On the flip side having resource shortages can indicate some parts of the economy are failing.

In general, graphs like these might let players know which professions need more active workers, and let admins know how healthy the server is.

**Comment by ThePiachu** [comment]
I guess some good stuff to track and graph would be:

- Total value of goods in stores (the currency page already guesstimates the value of one currency in comparison to another, so it wouldn't be a stretch to tally those up)
- Something relating to how much money is spent per day - GDP / velocity of money. Would probably be a good indicator for whether or not to print more money, etc.
- Maybe something about a median balance of an Active player? Some measurement of whether people have actual money in their account or is it concentrated in the hands of a few
- Some per-person tally of wealth (all the stuff their own, multiply by the price of the stuff) to measure how wealthy some people actually are if you take into consideration investments in transportation, housing, machinery, etc.

**Comment by johnkslg** [comment]
Yes! Would love to add all this stuff to the graphs.  Would need to make version for each currency (and nation) in the game
